### DISCUSSION TITLE
Feature Request: Alert Rule Evaluation Jitter

### DISCUSSION THREAD

**Discussion by Vimao** [grafana/grafana#71559] [Alerting]
I think that Grafana could benefit from the ability to setup alert evaluation jitter. Currently if the instance has a lot of alerting rules, they will be executed all at once. This causes CPU usage spikes on Grafana and datasources that could be smoothed out by not firing everything all at once. 

While global level jitter would work, i think jitter on an alert rule level would be even better, as we might require some rules to be precisely executed on specific times.

**Comment by yuri-tceretian** [comment]
Good idea! There is an issue https://github.com/grafana/grafana/issues/53744 where we discuss the implementation.

**Comment by gelicia** [comment]
Closing this discussion so the conversation can happen in the attached issue.
